api/config/v2alpha2/projectconfig_types.go

```diff
@@ -98,6 +98,50 @@ type ProjectConfig struct {
 	ActivityExporter  *ExporterConfig `json:"activityExporter,omitempty"`
 
 	PodRestart *PodRestartConfig `json:"podRestart,omitempty"`
+
+	// OpaControlPlaneConfig contains configuration for connecting to the
+	// OPA Control Plane APIs. If this is not set, the controller will not
+	// attempt to connect to the OPA Control Plane APIs.
+	OPAControlPlaneConfig *OPAControlPlaneConfig `json:"opaControlPlane,omitempty"`
+
+	// DefaultRequirements is a list of requirements that will be added to all
+	// systems created by the controller in the OCP, in addition to any requirements
+	// specified on the System resource.
+	DefaultRequirements []string `json:"defaultRequirements,omitempty"`
+
+	// ObjectStorage is the object storage configuration to use for bundles.
+	// currently only supports aws
+	ObjectStorage *ObjectStorage `json:"objectStorage,omitempty"`
+
+	// EnableStyraReconciliation is a flag that sets whether the controller should use Styra
+	// A Migration flag to enable/disable Styra DAS reconciliation for all systems and libraries.
+	EnableStyraReconciliation *bool `json:"enableStyraReconciliation,omitempty"`
```

Comment:
```suggestion
	EnableStyraReconciliation bool `json:"enableStyraReconciliation,omitempty"`
```
Dont think it needs to be a pointer to bool. This way we can remove all the nil checks.